resources/kubernetes/other-namespace.yaml

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
I'm wondering about three things regarding `other-namespace.yaml`:
* Wouldn't it be better to have a complete example? I.e. the same as is in the current single namespace example, with service account, deployment, Role etc., just with multiple bindings and multiple namespaces configured?
* Should this be references in _Getting started_? Or do we consider this too advanced? If this is too advanced, maybe we do not need this here.
* Do we need to add also OpenShift example? It might be confusing that this is only in the `kubernetes` folder.